### DISCUSSION TITLE
JSMpeg quality...

### DISCUSSION THREAD

**Discussion by cweakland** [blakeblackshear/frigate#2969] [Ask A Question] [answered]
I find that Frigate's JSMpeg streams to Home Assistant are really great in terms of stability and stream start times. Is there any way to increase the quality of the stream? It looks great on my phone, but on a larger monitor its a little lacking. 

Thank you for this project, it is amazing!

**Comment by NickM-27** [comment, marked as answer]
Check out the configuration file in docs: https://docs.frigate.video/configuration/index

![image](https://user-images.githubusercontent.com/14866235/158401372-4cc80b41-5dc4-47d4-aed4-545ef05677d1.jpeg)

**Reply by cweakland** [reply]
Amazing! Thank you Sir!

**Reply by cweakland** [reply]
Just wanted to come back and say this is perfect! 

I tried WebRTC on Home Assistant (native - Low Latency HLS and the addon ) the stability and load times gave me grief. However the JSMpeg streams on Home Assistant across platforms (IOS Companion App and Windows 10) strike the perfect balance between the stream delay, load times, stability, and quality. 

Thanks again!
